.travis.yml

```diff
@@ -20,7 +20,6 @@ script:
   - yarn lint
   - yarn tsc
   - yarn format
-  - yarn bundlesize --config bundlesize.config.js
```

Comment:
I didn't notice a bundlesize check for this PR. Would bundle size checks still be a part of the CI build process? Does it still use `bundlesize-store.now.sh`?

How does this change the developer workflow? Is the new workflow to manually compare new snapshot to previous snapshot on each commit rather than relying on `yarn bundlesize --config bundlesize.config.js`?

Reply:
The bundle size check is handled within Rollup when the `CI` environment variable is set. This follows the same process as https://github.com/zendeskgarden/react-containers/pull/158

The new workflow is that whenever a package is updated, the new size is reflected in the updated `.size-snapshot` file. These files are committed along with the PR so we can understand the bundle implications for every change.

Reply:
Awesome, sounds great!